File: cmd/umbra/bfc/mute.go
package bfc

import (
	"cast/lib"
	"os/exec"

	"github.com/spf13/cobra"
)

var MuteCmd = &cobra.Command{
	Use:   "mute",
	Short: "Mutes the default device",
	Run: func(cmd *cobra.Command, args []string) {
		cmdToRun := exec.Command("pactl", "set-sink-mute", "@DEFAULT_SINK@", "1")

		if err := cmdToRun.Run(); err != nil {
			lib.Logger.Fatal("Failed to mute", "err", err)
		}
	},
}

File: cmd/umbra/bfc/mute.py
import typer
import os

app = typer.Typer()

@app.command(rich_help_panel="Umbra")
def mute():
  """
  Mutes the default device
  """
  os.system("pactl set-sink-mute @DEFAULT_SINK@ 1")

#TODO Add flags for muting speaker and mic